### PR TITLE
feat: SR + CSM contracts proposed -> actual

### PR DESCRIPTION
**Pull request by dgusakov**
DO NOT MERGE unless the vote is enacted

### CODE REVIEW

**Review by avsetsin — APPROVED** [review]
lgtm

**Comment by TheDZhon** [comment]
:+1: 

One little concern: seems the CSM committee is missing here https://docs.lido.fi/multisigs/committees
cc: @lidofinance/lido-dao-ops-team

**Comment by dgusakov** [comment]
CSM Multisig was added to the MS page

@TheDZhon FYI

File: docs/multisigs/committees.md [hunk 1]
```diff
@@ -482,3 +482,43 @@ TopUpAllowedRecipients: [`0x1F2b79FE297B7098875930bBA6dd17068103897E`](https://
...
+cancel MEV stealing penalty if needed, starting EasyTracks to settle MEV stealing penalty, switching the bond curve for the particular Node Operator or resetting it to the default one,
+pausing CSM in case of emergency via Gate Seal.
+
+**Quorum:** 4/6
+
+**Forum topic:** [Community Staking Module Committee](https://research.lido.fi/t/csm-committee-creation/8333), [Community Staking Module](https://research.lido.fi/t/community-staking-module/5917)
+
+**Snapshot:** [Lido Community Staking Module Mainnet Release Setup](https://snapshot.org/#/lido-snapshot.eth/proposal/0xd0d7bfd68f2241524dbb14ae6fe0e8414b9fe3e0dcfc50641a8d28f0067d6693)
+
+**Aragon:** CSM was dded to the Lido on Ethereum protocol in [Omnibus vote #180 (items 20-26)](https://vote.lido.fi/vote/180)
```
Comment (avsetsin) [inline docs/multisigs/committees.md:501]:
```suggestion
**Aragon:** CSM was added to the Lido on Ethereum protocol in [Omnibus vote #180 (items 20-26)](https://vote.lido.fi/vote/180)
```

**Review by avsetsin — APPROVED** [review]

**Review by TheDZhon — APPROVED** [review]
LGTM

File: docs/deployed-contracts/index.md
Comment (TheDZhon) [inline docs/deployed-contracts/index.md:1]: I guess that the deployed `MinFirstAllocationStrategy` instance is missing :warning:

**Review by TheDZhon — CHANGES_REQUESTED** [review]

File: docs/deployed-contracts/index.md
Comment (avsetsin) [reply docs/deployed-contracts/index.md:1]: Fixed

**Review by TheDZhon — APPROVED** [review]
LGTM (what a bipolar man I am)

**Review by kate-aleksseeva — APPROVED** [review]
approved

**Pull request merged**